### ISSUE TITLE
[A11yMAS][Accessibility][High Contrast][MAS4.3.1][Eclipse][Wipro]:  For created project text  is clearly not visible for selected project in "Project Explorer" window in both highcontrast modes

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1451]
**Repro Steps:**
1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to "Project Explorer" and select required Project  then observe the selected project text in both hc modes

**Expected Result:**
For selected Azure project should be displayed in turquoise color background with black text in HC black mode and Violet color background with white text in HC white mode

**Actual Result:**
 For selected Azure Project are displayed as turquoise color background with white text in HC black mode and Violet color background with black text in HC white mode

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version:** Version 1709(Os Build 16299.192)
![for selected project is not following the highcontrast standards](https://user-images.githubusercontent.com/22956130/35790296-92d0bd88-0a67-11e8-80af-65adb0d94f8d.jpg)

**Comment by Eskibear** [comment]
It is determined by Eclipse IDE. We cannot fix it.

**Comment by Eskibear** [comment]
The color is as expected in:
```
Eclipse Java EE IDE for Web Developers.

Version: Oxygen.3a Release (4.7.3a)
Build id: 20180405-1200
```
Please help to re-verify.

![image](https://user-images.githubusercontent.com/2351748/43439935-024832b4-94c8-11e8-8904-0fdc4e161853.png)

**Comment by WiproPSAccTest** [comment]
Verified the issue in Version Oxygen 3a Release (4.7.3a) .Issue Still Repros .
![screenshot 3](https://user-images.githubusercontent.com/22956130/43779837-ec34c226-9a76-11e8-853e-2d168678e6e9.png)
![screenshot 5](https://user-images.githubusercontent.com/22956130/43779839-ec658154-9a76-11e8-967b-4411b48bdad9.png)

**Comment by WiproPSAccTest** [comment]
Verified the issue in Version Oxygen 3a Release (4.7.3a)
OS Version - 1803 (OS Build 17134.167)
 .Issue Still Repro .

**Comment by Eskibear** [comment]
* **Importance of experience**: minimal. It's not a valid use case. 
* **Quality of experience**: not impacted. Users are not required to open this dialog in any scenario. 
* **Alternate solutions**: No alternate technologies or options can be leveraged.
* **Evidence**: All evidence is provided in this issue and external issue.

**Comment by umasarath52** [comment]
Discussed with Ryan and approved to close as this scenario is non blocking and low impact.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749888
